src/generate-note-controller.js

```diff
@@ -114,9 +185,12 @@ module.exports = function(scribe){
         var selectionIsCollapsed = (markers.length === 1);
 
         //we need to figure out if our caret or selection is within a conflicting note
-        var isWithinConflictingNote = config.get('selectors').reduce((last, selector)=>{
-          return selector.tagName !== tagName ? !!isSelectionWithinNote(markers[0], selector.tagName) : last;
-        }, false);
+        var isWithinConflictingNote = false;
+        config.get('selectors').forEach((selector)=>{
+          if((selector.tagName !== tagName) && isSelectionWithinNote(markers, selector.tagName)){
+            isWithinConflictingNote = true;
+          }
+        });
```

Comment:
Or even better than filter (or forEach), [`Array.prototype.some`](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/some) (should only walk the array until true is returned, so O(n) worst case rather than always).